### PR TITLE
ADD README.md and docblock

### PR DESCRIPTION
**Pull request by MayMeow**
I add README.md file and docblocks for classes.

### CODE REVIEW

File: src/Exceptions/HashFailedException.php [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 namespace Dpc\HashVerifier\Exceptions;
 
-
+/**
+ * Class HashFailedException
+ * @package Dpc\HashVerifier\Exceptions
```
Comment (Dylan-DPC-zz) [inline src/Exceptions/HashFailedException.php]: this isn't needed. It is understood from the namespace

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Fork it!
+2. Create your feature branch: `git checkout -b my-new-feature`
+3. Commit your changes: `git commit -am 'Add some feature'`
+4. Push to the branch: `git push origin my-new-feature`
+5. Submit a pull request :D
+
+## History
+
+TODO: Write history
```
Comment (Dylan-DPC-zz) [inline README.md]: history isn't needed

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Installation
+
+```bash
+composer require dpc/hash-verifier
+```
+
+## Usage
+
+TODO: Write usage instructions
```
Comment (Dylan-DPC-zz) [inline README.md]: can you attempt this based on the example I have given? If not I will add it later

File: .idea/modules.xml [hunk 4]
```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/Authvalidator.iml" filepath="$PROJECT_DIR$/.idea/Authvalidator.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/hash-verifier.iml" filepath="$PROJECT_DIR$/.idea/hash-verifier.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/hash-verifier.iml" filepath="$PROJECT_DIR$/.idea/hash-verifier.iml" />
```
Comment (Dylan-DPC-zz) [inline .idea/modules.xml]: can you remove this folder from tracking? had added it by mistake

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

File: src/Exceptions/HashFailedException.php (hunk 1 above)
Comment (MayMeow) [reply src/Exceptions/HashFailedException.php]: ok ill remove it

**Comment by MayMeow** [comment]
Yes because you have tracked /.idea/* folder ... ;)

**Comment by MayMeow** [comment]
.. and my docblcks :D

**Review by Dylan-DPC-zz — APPROVED** [review]

**Pull request merged**